build/media-suite_compile.sh

```diff
@@ -1227,6 +1227,22 @@ if [[ $mp4box = "y" ]] && do_vcs "https://github.com/gpac/gpac.git"; then
     do_checkIfExist
 fi
 
+_check=(SvtHevcEnc.pc libSvtHevcEnc.dll{,.a} svt-hevc/EbApi.h)
+if [[ $svthevc = "y" ]] && do_vcs "https://github.com/intel/SVT-HEVC.git"; then
+    [[ $standalone = "y" ]] && _check+=(SvtHevcEncApp.exe)
+    do_uninstall "${_check[@]}"
+    ORIGINAL_PKG_CFG=$PKG_CONFIG
+    PKG_CONFIG="$LOCALDESTDIR/bin/ab-pkg-config-static.bat"
+    do_patch
```

Comment:
Ignore this for now, leaving until I can make a proper patch to replace march=native.